### PR TITLE
Add troubleshooting guide for the registration issue at deployment

### LINKED ISSUE #319: Add a troubleshooting guide for authentication issues
Add to the documentation the steps to fix authentication issues during deployments

### PR DESCRIPTION
**Pull request by chkeita**

### CODE REVIEW

**Review by ranweiler — APPROVED** [review]

File: docs/known-issues.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
    This means someone already is using `myname.azurewebsites.net`. You'll need
    to pick a different name for your onefuzz instance.
 
-1. The workspace name 'logs-wb-XXX' is not unique
+2. The workspace name 'logs-wb-XXX' is not unique
```
[the remark this answers is not in the document]
Comment (bmc-msft) [reply docs/known-issues.md:10]: The only issue with doing that is the markdown itself isn't as readable.   Most of the documentation in OneFuzz uses the repeated "1." style, though in writing the declarative template management guide, I found it to be distracting.

[the remark this answers is not in the document]
Comment (bmc-msft) [reply docs/known-issues.md:10]: @ranweiler @chkeita , do either you have strong preferences for this?  Whatever is chosen, we should be consistent and update the docs to match the style.

[the remark this answers is not in the document]
Comment (ranweiler) [reply docs/known-issues.md:10]: I don't care too much one way or the other, and would be fine with saying "we should maintain correct order list numberings in markdown source".

That said, here we "fix" just one random item, then add a new `1.` item (many items) later.

[the remark this answers is not in the document]
Comment (chkeita) [reply docs/known-issues.md:10]: I personally prefer having the correct ordering in the markdown. I will update the other ones in this document

File: docs/known-issues.md [hunk 2]
```diff
@@ -7,26 +7,46 @@
...
-1. Azure.Functions.Cli.Arm.ArmResourceNotFoundException: Can't find app with
+4. Azure.Functions.Cli.Arm.ArmResourceNotFoundException: Can't find app with
    name "XXX"
 
    The resources for the onefuzz instance were deployed, but the SCM component
    of Azure Functions was not available yet. This race condition is solved by
    ARM reporting the deployment is finished too early. Retry the deployment and
    the error should be corrected automatically.
+
+5. Registration.GraphQueryError: request did not succeed: HTTP 403
```
Comment (chkeita) [inline docs/known-issues.md:34]: will add another issue related to the resource group

**Pull request merged**